storage/block/block.go

```diff
@@ -56,10 +56,24 @@ type dbBlock struct {
 	retriever  DatabaseShardBlockRetriever
 	retrieveID ident.ID
 
+	owner OnEvictedFromWiredList
```

Comment:
nit: I would rename `owner` to `onEvicted` if I was making this change, just to make it consistent *shrug*.

Reply:
yeah good call I missed this during my rename